README.md

```diff
@@ -207,20 +207,24 @@ Current time step length: 0.050000
  ────────────────────────────────────────────────────────────────────────────────────
 ```
 
-If you change the executable name from `simple_trixi_controller_c` to
-`simple_trixi_controller_f`, you will get a near identical output. The corresponding source
-files `simple_trixi_controller.c` and `simple_trixi_controller.f90` give you an idea on how
-to use the C and Fortran APIs of libtrixi, and can be found in the
-[`examples/`](examples/) folder.
+If you change the executable name from `trixi_controller_simple_c` to
+`trixi_controller_simple_f`, you will get a near identical output. The corresponding source
+files can be found in the [`examples/`](examples/) folder. The examples demonstrate different
+aspects on how to use the C and Fortran APIs of libtrixi:
+
+- `trixi_controller_simple.(c|f90)`: basic usage
+- `trixi_controller_mpi.(c|f90)`: usage in the presence of MPI
+- `trixi_controller_data.(c|f90)`: simulation data access
+- `trixi_controller_t8code.c`: interacting with t8code
```

Comment:
Indeed, when I added the functionality in the first place I could not think of a Fortran example because the t8code Fortran interface was still under development. That could have change in the meantime however. I will have another look.

Reply:
It has not changed.

Reply:
I added a comment.